src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/NewAzureRmVmssConfigCommand.cs

```diff
@@ -399,6 +412,24 @@ private void Run()
                 vSku.Name = this.SkuName;
             }
 
+            if (this.IsParameterBound(c => c.EnableResilientVMCreate))
```

Comment:
for switch parameter checks, we've been using "this.ParameterName == True" because users can still pass in "-EnableResilientVMCreate $false" and this line will pass in true

Reply:
resolved offline. If user uses -EnableResilientVMCreate $false then they have intention of using the property. So AFEC flag check is fine. Resolving this.